test/sqllogictest/explain/physical_plan_as_text.slt

```diff
@@ -776,28 +776,79 @@ EXPLAIN PHYSICAL PLAN AS TEXT FOR
 MATERIALIZED VIEW collated_group_by_mv
 ----
 materialize.public.collated_group_by_mv:
-  Reduce::Collation
```

Comment:
Could you adjust the comment at the top of this test?

Also, when we have a feature flag for the new transform, maybe we could have two copies of this test, where one copy is with the feature flag off and one is with on.

Reply:
Can we just delete the test instead? If there is no longer a `Reduce::Collated` to test, no need to keep the test.

Reply:
Well, there will be, with the feature flag off.

What's the plan for `Reduce::Collated`? Will it be deleted (together with the feature flag) a few weeks from now when we have verified that no customers have any serious regressions?